src/Tests/InstrEngineTests/InstrEngineTests/TargetAppCompiler.cs

```diff
@@ -38,9 +38,7 @@ internal class TargetAppCompiler
         private const string X86BinarySuffix = "x86";
         private const string X64BinarySuffix = "x64";
 
-        private const string EmbeddedResourcesPath = "InstrEngineTests.EmbeddedResources";
-
-        private static string[] TestAppFilePrefixes = new string[] {
+        private static string[] EmbeddedCSharpFiles = new string[] {
```

Comment:
nit: These are still prefixes (same with `EmbeddedILFiles`), so it might be good to keep it in the name (e.g. `EmbeddedCSharpFilePrefixes`)